libs/martech/attributes.js

```diff
@@ -1,13 +1,66 @@
-const RE_ALPHANUM = /[^0-9a-z]/gi;
-const RE_TRIM_UNDERSCORE = /^_+|_+$/g;
+export function processTrackingLabels(text, config, charLimit = 20) {
+  if (!config) {
+    import('../utils/utils.js').then((utils) => {
+      // eslint-disable-next-line no-param-reassign
+      config = utils.getConfig();
+    });
+  }
+  const analyticsValue = text?.replace(/[^\w\s]+/g, ' ').replace(/\s+/g, ' ').replace(/^_+|_+$/g, '').trim()
+    .slice(0, charLimit);
+  return analyticsValue;
+}
 
-// decorateBlockAnalytics & decorateLinkAnalytics are being sunset
+export function decorateDefaultLinkAnalytics(block, config) {
+  if (block.classList.length
+    && !block.className.includes('metadata')
+    && !block.classList.contains('link-block')
```

Comment:
Why `className` && `classList`?  Are you looking for any class containing `metadata`?

Reply:
line 15 is className to catch different metadata blocks: section-metadata, table-metadata etc. I think this future proofs it more
line 16 is classList because the exact name of the class I'm checking for is "link-block"